### DISCUSSION TITLE
Should we add benchmark to encourage developers upgrade to latest version?

### DISCUSSION THREAD

**Discussion by gweesin** [verdaccio/verdaccio#4570] [Ideas]
My company is still using verdaccio v4, and the leader doesn't want to upgrade verdaccio because he could not know it's advantages. I think that's good if we can add a benchmark to encourage developers convince their leader bump the version.

**Comment by juanpicado** [comment]
I tried but I discontinued the idea for lack of time 

https://github.com/juanpicado/benchmark

One reason for your company is upgrade because I don't maintain verdaccio 4 anymore and is not getting any security or dependencies updates, only v5 and ahead.

I'd imagine is a good reason but yeah every company is a different world.

**Reply by gweesin** [reply]
Yes, benchmarks will be more convincing, but security is also one aspect. Thank you for your contribution to open source, thank you for your efforts!
